setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name="Open edX",
-    version="0.3",
+    version="0.3.1",
```

Comment:
We should work out the version scheme to use for edX platform. This doesn't seem like a patch change to me, but the minor version is going to ramp quickly if every feature bumps it.